drivers/serial/uart_xlnx_ps.c

```diff
@@ -34,38 +34,43 @@
+#define XUARTPS_CR_OFFSET		0x0000U  /**< Control Register [8:0] */
+#define XUARTPS_MR_OFFSET		0x0004U  /**< Mode Register [9:0] */
+#define XUARTPS_IER_OFFSET		0x0008U  /**< Interrupt Enable [12:0] */
+#define XUARTPS_IDR_OFFSET		0x000CU  /**< Interrupt Disable [12:0] */
+#define XUARTPS_IMR_OFFSET		0x0010U  /**< Interrupt Mask [12:0] */
+#define XUARTPS_ISR_OFFSET		0x0014U  /**< Interrupt Status [12:0]*/
 #define XUARTPS_BAUDGEN_OFFSET	0x0018U  /**< Baud Rate Generator [15:0] */
 #define XUARTPS_RXTOUT_OFFSET	0x001CU  /**< RX Timeout [7:0] */
-#define XUARTPS_RXWM_OFFSET	0x0020U  /**< RX FIFO Trigger Level [5:0] */
+#define XUARTPS_RXWM_OFFSET		0x0020U  /**< RX FIFO Trigger Level [5:0] */
 #define XUARTPS_MODEMCR_OFFSET	0x0024U  /**< Modem Control [5:0] */
 #define XUARTPS_MODEMSR_OFFSET	0x0028U  /**< Modem Status [8:0] */
-#define XUARTPS_SR_OFFSET	0x002CU  /**< Channel Status [14:0] */
-#define XUARTPS_FIFO_OFFSET	0x0030U  /**< FIFO [7:0] */
+#define XUARTPS_SR_OFFSET		0x002CU  /**< Channel Status [14:0] */
+#define XUARTPS_FIFO_OFFSET		0x0030U  /**< FIFO [7:0] */
 #define XUARTPS_BAUDDIV_OFFSET	0x0034U  /**< Baud Rate Divider [7:0] */
 #define XUARTPS_FLOWDEL_OFFSET	0x0038U  /**< Flow Delay [5:0] */
-#define XUARTPS_TXWM_OFFSET	0x0044U  /**< TX FIFO Trigger Level [5:0] */
-#define XUARTPS_RXBS_OFFSET	0x0048U  /**< RX FIFO Byte Status [11:0] */
+#define XUARTPS_TXWM_OFFSET		0x0044U  /**< TX FIFO Trigger Level [5:0] */
+#define XUARTPS_RXBS_OFFSET		0x0048U  /**< RX FIFO Byte Status [11:0] */
 
 /* Control Register Bits Definition */
-#define XUARTPS_CR_STOPBRK	0x00000100U  /**< Stop transmission of break */
-#define XUARTPS_CR_STARTBRK	0x00000080U  /**< Set break */
-#define XUARTPS_CR_TORST	0x00000040U  /**< RX timeout counter restart */
-#define XUARTPS_CR_TX_DIS	0x00000020U  /**< TX disabled. */
-#define XUARTPS_CR_TX_EN	0x00000010U  /**< TX enabled */
-#define XUARTPS_CR_RX_DIS	0x00000008U  /**< RX disabled. */
-#define XUARTPS_CR_RX_EN	0x00000004U  /**< RX enabled */
+#define XUARTPS_CR_STOPBRK		0x00000100U  /**< Stop transmission of break */
+#define XUARTPS_CR_STARTBRK		0x00000080U  /**< Set break */
+#define XUARTPS_CR_TORST		0x00000040U  /**< RX timeout counter restart */
+#define XUARTPS_CR_TX_DIS		0x00000020U  /**< TX disabled. */
+#define XUARTPS_CR_TX_EN		0x00000010U  /**< TX enabled */
+#define XUARTPS_CR_RX_DIS		0x00000008U  /**< RX disabled. */
+#define XUARTPS_CR_RX_EN		0x00000004U  /**< RX enabled */
 #define XUARTPS_CR_EN_DIS_MASK	0x0000003CU  /**< Enable/disable Mask */
-#define XUARTPS_CR_TXRST	0x00000002U  /**< TX logic reset */
-#define XUARTPS_CR_RXRST	0x00000001U  /**< RX logic reset */
+#define XUARTPS_CR_TXRST		0x00000002U  /**< TX logic reset */
+#define XUARTPS_CR_RXRST		0x00000001U  /**< RX logic reset */
 
 /* Mode Register Bits Definition */
-#define XUARTPS_MR_CCLK			0x00000400U /**< Input clock select */
+#define XUARTPS_MR_CCLK				0x00000400U /**< Input clock select */
```

Comment:
Nit: The standard TAB width for Zephyr is 8 characters (see `.editorconfig`); please set alignments based on this.